app/views/play/CampaignView.coffee

```diff
@@ -196,7 +196,9 @@ module.exports = class CampaignView extends RootView
     context.requiresSubscription = @requiresSubscription
     context.editorMode = @editorMode
     context.adjacentCampaigns = _.filter _.values(_.cloneDeep(@campaign?.get('adjacentCampaigns') or {})), (ac) =>
-      return false if ac.showIfUnlocked and (ac.showIfUnlocked not in me.levels()) and not @editorMode
+      if ac.showIfUnlocked and not @editorMode
+        return false if _.isString ac.showIfUnlocked and ac.showIfUnlocked not in me.levels()
```

Comment:
Need some parentheses here or _.isString is going to apply to the whole `and`, and same in next line